test/test_point.py

```diff
@@ -1,72 +1,81 @@
 import unittest
 
-from src.point import Point
+from src.point3d import Point3D
 
 
 class TestPoint(unittest.TestCase):
-    def test_point_0_0_has_x_0(self):
-        point = Point(0, 0)
+    def test_point_0_0_0_has_x_0(self):
+        point = Point3D(0, 0, 0)
         self.assertEqual(0, point.x)
 
-    def test_point_0_0_has_y_0(self):
-        point = Point(0, 0)
+    def test_point_0_0_0_has_y_0(self):
+        point = Point3D(0, 0, 0)
         self.assertEqual(0, point.y)
 
+    def test_point_0_0_0_has_z_0(self):
+        point = Point3D(0, 0, 0)
+        self.assertEqual(0, point.z)
+
     def test_update_point_x_has_updated_x(self):
-        point = Point(0, 0)
+        point = Point3D(0, 0, 0)
         point.x = -1
         self.assertEqual(-1, point.x)
 
     def test_update_point_y_has_updated_y(self):
```

Comment:
What are you testing with these? This is basically a language feature.

Reply:
Original thought --- _overkill will be good for having them think about testing thorough_ 
now --- _no, that's bad_

They were yeeted with the latter class tests, idk why I left them here 🤷